docs/graphql/manual/engine-internals/index.rst

```diff
@@ -11,7 +11,7 @@ initializes a few tables under it as described below.
 hdb_catalog
 -----------
 
-The schema created by Hasura GraphQL engine where it manages its internal state. Whenever a
+The schema is created by Hasura GraphQL engine where it manages its internal state. Whenever a
```

Comment:
"This schema is created by Hasura GraphQL Engine to manages its internal state."